client/package.json

```diff
@@ -52,7 +52,7 @@
     "sass-loader": "^8.0.0",
     "sinon": "^8.1.1",
     "style-resources-loader": "^1.3.2",
-    "typescript": "^3.5.3",
+    "typescript": "^3.8.3",
```

Comment:
It's actually already on v3.9.3.

Reply:
Ya, i don't understand this one, how can the package and lock be so out of sync.

Reply:
See my response above. It's due to the caret.

(And that's the reason we don't use a caret for the sbc-common-components.)